core/geometry/src/polyface/PolyfaceBuilder.ts

```diff
@@ -59,6 +59,8 @@ import { IndexedPolyfaceSubsetVisitor } from "./IndexedPolyfaceVisitor";
 import { IndexedPolyface, PolyfaceVisitor } from "./Polyface";
 import { PolyfaceQuery } from "./PolyfaceQuery";
 
+/* eslint-disable deprecation/deprecation */
```

Comment:
If there eslint-disable syntax that targets only the next/current line, that's a more preferable to the current syntax, which disables deprecation warnings throughout the file.

Reply:
I tried that. Did not work.

Reply:
You tried this line _directly above_ the recursive call on line 1227?

// eslint-disable-next-line deprecation/deprecation

Reply:
No, line 1219

![image](https://github.com/iTwin/itwinjs-core/assets/114527638/f6c21c38-29c7-4663-b1e5-948da205d8b3)

Reply:
line 1227 works. Will make the change and commit